src/main/java/mesosphere/marathon/client/model/v2/App.java

```diff
@@ -41,7 +41,7 @@ public String toString() {
 	private Container container;
 	private Map<String, Object> env;
 	private Map<String, String> labels;
-	private AppIpAddress ipAddress;
+	private IpAddress ipAddress;
```

Comment:
I rebased off of Mesosphere's master branch of the client, and they had made this change.